cmd/podman/volumes/export.go

```diff
@@ -0,0 +1,82 @@
+package volumes
+
+import (
+	"context"
+
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/inspect"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/libpod/define"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/utils"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumeExportDescription = `
+podman volume export
+
+Allow content of volume to be exported into external tar.`
+	exportCommand = &cobra.Command{
+		Use:               "export [options]",
+		Aliases:           []string{"export"},
+		Short:             "Export volumes",
+		Long:              volumeExportDescription,
+		RunE:              export,
+		ValidArgsFunction: common.AutocompleteVolumes,
+	}
+)
+
+var (
+	// Temporary struct to hold cli values.
+	cliExportOpts = struct {
+		Output string
+	}{}
+	exportOpts = entities.VolumeListOptions{}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: exportCommand,
+		Parent:  volumeCmd,
+	})
+	flags := exportCommand.Flags()
+
+	outputFlagName := "output"
+	flags.StringVar(&cliExportOpts.Output, outputFlagName, "", "Output=[path]")
+	_ = exportCommand.RegisterFlagCompletionFunc(outputFlagName, common.AutocompleteFormat(define.InspectVolumeData{}))
```

Comment:
This does not make sense. Output accepts a path so this should use `AutocompleteDefault` and not a completion function for a go template.

Reply:
@Luap99 Afaik this is need in-oder to autocomplete `volume names`. But i will confirm.